### ISSUE TITLE
Review and update website page (https://www.tdwg.org/community/cd/)

### ISSUE THREAD

**Issue by mswoodburn** [tdwg/cd#13]

**Comment by debpaul** [comment]
This is the IG Charter essentially. Am I right @stanblum @baskaufs? 

Am I correct in that this page is:

- [ ] generated from the gh-pages branch
- [ ] i can clone the gh-pages repo and then edit and make a pull request

**Comment by stanblum** [comment]
If I remember correctly, this is the page where the IG members can craft and finalize, then once it's ready, the web maintainers can copy this text and put it into place on the website.  So, this is not live to the website, but can be copied to the website when ready.  If that's too much trouble for the current state of affairs (only minor edits needed), those can be made directly on the website by creating a fork, editing, and then making a pull request to have those edits merged into the website master branch.  See the "edit this page" link on the CD landing page (https://www.tdwg.org/community/cd/), and also the website-faq page under TDWG / About (https://www.tdwg.org/about/website-faq/

**Comment by debpaul** [comment]
Yes, I am going to do the latter on the interest of time. We have consensus on what we want / need to do.

Thanks Stan, Deb
